flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicPartitionStateWithPeriodicWatermarks.java

```diff
@@ -61,6 +61,10 @@ public long getCurrentWatermarkTimestamp() {
 		return partitionWatermark;
 	}
 
+	void setCurrentWatermarkTimestamp(long watermarkTimestamp) {
```

Comment:
The other methods seem to be `public` (although they can actually be package-private). Should we stay consistent with that here?